boards.txt

```diff
@@ -4865,6 +4865,15 @@ GenF4.menu.pnum.GENERIC_F407ZGTX.build.product_line=STM32F407xx
 GenF4.menu.pnum.GENERIC_F407ZGTX.build.variant=STM32F4xx/F407Z(E-G)T_F417Z(E-G)T
 GenF4.menu.pnum.GENERIC_F407ZGTX.debug.svd_file={runtime.tools.STM32_SVD.path}/svd/STM32F4xx/STM32F407.svd
 
+# Generic F407IGHx
```

Comment:
When adding a generic variant all all possible boards entries have to be added in baords.txt by alphabetical order.
It misses:
Generic F407IEHx
Generic F407IETx
Generic F407IGTx
Generic F417IEHx
Generic F417IGHx
Generic F417IETx
Generic F417IGTx

Reply:
Done